src/Tools/Md2Html/Md2Html.csproj

```diff
@@ -69,4 +74,21 @@
     </PackageReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="AfterBuild">
+    <!-- the ILMergePath property points to the location of ILMerge.exe console application -->
+    <Exec Command="$(ILMergeConsolePath) $(OutputPath)Md2Html.exe ^
```

Comment:
where does this live though? In the packages folder?

Reply:
Yes, that just works.